install/helm/agones/templates/serviceaccounts/sdk.yaml

```diff
@@ -38,7 +38,7 @@ metadata:
 rules:
 - apiGroups: ["stable.agones.dev"]
   resources: ["gameservers"]
-  verbs: ["get", "update"]
+  verbs: ["get", "list", "update"]
```

Comment:
Do we still need get since you have a cache sdk side with the list ?

Reply:
Nicely spotted! No we do not. Removed `get`.